src/scripts/__tests__/__snapshots__/format.js.snap

```diff
@@ -1,9 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`format --config arg can be used for a custom config 1`] = `prettier --ignore-path ./src/config/prettierignore --write **/*.+(js|jsx|json|less|css|ts|tsx) --config ./my-config.js`;
+exports[`format --config arg can be used for a custom config 1`] = `prettier --ignore-path ./src/config/prettierignore --write **/*.+(js|json|less|css|ts|tsx|md) --config ./my-config.js`;
```

Comment:
I wonder why jsx files are not included?

Reply:
I think `tsx` is only there because of the typescript convention of only parsing jsx syntax if there's a silly `x` on the filename.